plugins/mcreator-localization/help/default/plant/static_generator_type.md

```diff
@@ -1,4 +0,0 @@
-How your static plant will be generated. 

```

Comment:
If this file is no longer used, delete it, not empty.

Reply:
Wasn't it deleted?
![image](https://user-images.githubusercontent.com/71347607/97952594-07117900-1daf-11eb-813e-3963c040bdc1.png)

Reply:
Oh ok, it showed as emptied, not removed to me yesterday.